src/main/resources/static/js/query.js

```diff
@@ -34,4 +56,18 @@ function deleteDisease(name) {
             $('#get-disease').click();
         }
     });
-}
\ No newline at end of file
+}
+
+function populateTreatment(data) {
+  $('#treatment-table tbody').remove();
+  let rows = "";
+  for(const row of data){
+    rows += `<tr><td>${row.treatmentName}</td>
+                 <td>${row.efficiency}</td>
+                 <td>${row.equipment}</td>
+                 <td>${row.risks}</td>
+                 <td><a class="btn btn-info" href="https://en.wikipedia.org/wiki/${row.treatmentName}"> More Info</a></td></tr>`;
```

Comment:
Nice:)

Reply:
oh haha well will be fun when we demo it